app/components/ui/Card.tsx

```
@@ -4,10 +4,15 @@ import styles from "./Card.module.css";
 interface CardProps {
   children: ReactNode;
   className?: string;
+  onClick?: () => void;
 }
 
-export function Card({ children, className = "" }: CardProps) {
-  return <div className={`${styles.card} ${className}`.trim()}>{children}</div>;
+export function Card({ children, className = "", onClick }: CardProps) {
+  return (
+    <div className={`${styles.card} ${className}`.trim()} onClick={onClick}>
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)